kuka_lbr_iiwa_support/CMakeLists.txt

```diff
@@ -5,7 +5,7 @@ find_package(catkin REQUIRED)
 
 catkin_package()
 
-foreach(dir config launch meshes/lbr_iiwa_14_r820/collision meshes/lbr_iiwa_14_r820/visual urdf) 
+foreach(dir config launch meshes urdf) 
```

Comment:
We can do this later for the packages in this repository as well. I'd like to avoid making PRs too 'big', so lets leave that for later.